aiotg/bot.py

```diff
@@ -220,6 +220,15 @@ def group(self, group_id):
             logger.error(err_msg)
             raise RuntimeError(err_msg)
 
+    _get_me = partialmethod(api_call, "getMe")
+
+    def get_me(self):
+        """
+        Returns basic information about the bot
+        (see https://core.telegram.org/bots/api#getme)
+        """
+        return self._get_me()
+
```

Comment:
I guess partialmethod is not necessary here, could you remove _get_me and instead, use api_call directly from get_me()?